Vagrantfile

```diff
@@ -29,13 +31,15 @@ Vagrant.configure(2) do |config|
       nasm \
       psmisc \
       qemu \
+      gcc-arm-none-eabi \
+      xorriso \
       rake
 
     # The gcc-arm-none-eabi package is not yet available in stretch:
     # https://packages.debian.org/jessie/devel/gcc-arm-none-eabi
-    echo deb http://httpredir.debian.org/debian/ unstable main >> /etc/apt/sources.list
-    sudo apt-get update
-    sudo DEBIAN_FRONTEND=noninteractive apt-get install -y gcc-arm-none-eabi
+    #echo deb http://httpredir.debian.org/debian/ unstable main >> /etc/apt/sources.list
+    #sudo apt-get update
+    #sudo DEBIAN_FRONTEND=noninteractive apt-get install -y gcc-arm-none-eabi
```

Comment:
Perhaps we can remove both the commented out code and the comment before? Since the package is now available (in newer Debian versions).